examples/template_alignment.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+"""Creating a template for a group of subject using alignment
+===================================================================
+
+In this tutorial, we show how to better predict new contrasts for a target
+subject using many source subjects corresponding contrasts. For this purpose,
+we create and template to which we align the target subject.
+
+We mostly rely on python common packages and on nilearn to handle
+functional data in a clean fashion.
+
+
+To run this example, you must launch IPython via ``ipython
+--matplotlib`` in a terminal, or use ``jupyter-notebook``.
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+"""
+
+# Retrieving the data
+# -------------------
+# In this example we use the IBC dataset, which include a large number of \
+# different contrasts maps for 12 subjects.
+# We download the images for subjects 1 2, 4, 5, 6 and 7 (or retrieve them \
+# if they were already downloaded).
+# Files is the list of paths for each subjects.
+# df is a dataframe with metadata about each of them.
+# mask is an appropriate nifti image to select the data.
+#
+
+from fmralign.fetch_example_data import fetch_ibc_subjects_contrasts
+files, df, mask = fetch_ibc_subjects_contrasts(
+    ['sub-01', 'sub-02', 'sub-04', 'sub-05', 'sub-06', 'sub-07'])
+
+###############################################################################
+# Defining a masker
+# -----------------
+# Using the mask provided, define a nilearn masker that will be used to \
+# handle relevant data. For more information, visit :
+# http://nilearn.github.io/manipulating_images/masker_objects.html
+#
+
+from nilearn.input_data import NiftiMasker
+masker = NiftiMasker(mask_img=mask)
+mask
+masker.fit()
+
+###############################################################################
+# Split the data
+# ---------------------------------------------
+# For each subject, we will use two 'copies' of data acquired in the same \
```

Comment:
I don't like this wording: this gives the imrpession that hyperalignment only works if you have 2 copies of the data.
Just say two different series of contrast (we never explicitly use the fact that there is redundancy in the contrast specification).